docs/swagger.yaml

```diff
@@ -493,7 +640,54 @@ components:
       schema:
         type: string
       description: Bearer <MSI token issued to the identity of the client>
+    Transaction-Type:
+      name: type
+      in: path
+      required: true
+      schema:
+        type: string
+        enum:
+          - ecom
+          - recurring
+          - checkout
```

Comment:
Lets keep checkout out for now, we'll have to have a talk with Alf magnus first to figure out what orderId they should use. Maybe even let them continue using v1